docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Server_Faces.adoc

```diff
@@ -1,163 +0,0 @@
-[[Jakarta_Server_Faces]]
-= Jakarta Server Faces Configuration
-
-[abstract]
-
-Jakarta Server Faces configuration is handled by the Jakarta Server Faces subsystem. The Jakarta Server Faces subsystem
-allows multiple Jakarta Server Faces implementations to be installed on the same WildFly
-server. In particular, any version of Mojarra or MyFaces that implements
-spec level 2.1 or higher can be installed. For each Jakarta Server Faces implementation,
-a new slot needs to be created under `com.sun.jsf-impl`,
-`jakarta.faces.api`, and `org.jboss.as.jsf-injection`. When the Jakarta Server Faces
-subsystem starts up, it scans the module path to find all of the Jakarta Server Faces
-implementations that have been installed. The default Jakarta Server Faces implementation
-that WildFly should use is defined by the `default-jsf-impl-slot`
-attribute.
-
-[[installing-a-new-jsf-implementation-manually]]
-== Installing a new Jakarta Server Faces implementation manually
-
-A new Jakarta Server Faces implementation can be manually installed as follows:
-
-[[add-a-module-slot-for-the-new-jsf-implementation-jar]]
-=== Add a module slot for the new Jakarta Server Faces implementation JAR
-
-* Create the following directory structure under the
-WILDFLY_HOME/modules directory: +
-WILDFLY_HOME/modules/com/sun/jsf-impl/<JSF_IMPL_NAME>-<JSF_VERSION> +
- +
-For example, for Mojarra 2.2.11, the above path would resolve to: +
-WILDFLY_HOME/modules/com/sun/jsf-impl/mojarra-2.2.11
-
-* Place the Jakarta Server Faces implementation JAR in the <JSF_IMPL_NAME>-<JSF_VERSION>
-subdirectory. In the same subdirectory, add a `module.xml` file similar
-to the
-https://github.com/wildfly/wildfly/blob/main/jsf/multi-jsf-installer/src/main/resources/mojarra-impl-module.xml[Mojarra]
```

Comment:
It's ok to remove the documentation here for now since the example module.xml files referenced here are out of date anyway since they only pertain to manully installing JSF 2.1.x or 2.2.x versions. However, I think it would be good to get a WFLY issue created for adding back this documentation and updating it. Documenting the ability to manually install a JSF implementation doesn't depend on the multi-JSF installer work being done first. We just need to have some example module.xml files that have been updated with the correct dependencies for JSF 3.x.

Reply:
https://issues.redhat.com/browse/WFLY-17394